requirements.txt

```diff
@@ -1,5 +1,5 @@
-numpy==1.17.0
-pandas==0.25.0
+numpy==1.17.5
+pandas==0.25.3
```

Comment:
@fschloesser , did you test pandas 1.x? Any incompatibilities?

Reply:
I didn't test anything on my machine, that is why i only updated the patch-versions. 

Thankfully we have the tests and could just trying to bump the versions and see what travis says. Although skimming over the output i saw a deprecation warning and a future warning. Haven't looked into it yet though out of time issues

Reply:
Okay. Let me know if you need help.